ros_cross_compile/ros_cross_compile.py

```diff
@@ -116,6 +117,13 @@ def parse_args(args: List[str]) -> argparse.Namespace:
         help='Relative path within the workspace to a file that provides colcon arguments. '
              'See "Package Selection and Build Customization" in README.md for more details.')
 
+    parser.add_argument(
+        '--skip-rosdep-collection',
+        action='store_true',
+        required=False,
+        help='User can skip collecting ros dependencies if the initial dependencies'
```

Comment:
I'd like to reword this a little bit. How about

"Skip querying rosdep for dependencies. This is intended to save time when running repeatedly during development, but has undefined behavior if the workspace's dependencies have changed since the last time collection was run."